src/Ormolu/Printer/Combinators.hs

```diff
@@ -245,6 +246,13 @@ braces m = sitcc $ do
   ospaces m
   txt "}"
 
+-- | Surround given entity by parallel curly braces @{:@ and @:}@.
+bracesPar :: R () -> R ()
```

Comment:
Why is this called `bracesPar` and says in the documentation that it surrounds `m` with `{:` and `:}` while it clearly just uses `{` and `}`. AFAIU that you also want normal braces for what you're trying to fix in this PR.

Reply:
That's an error. I'll fix it now. Thanks!